### PR TITLE
[#WOFFICE-15689] add loco client

### PR DESCRIPTION
**Pull request by AGoncharenko**

### CODE REVIEW

**Comment by xiaofenggh15** [comment]
looks good

**Comment by mahesh-karipe** [comment]
Ruby client for Localize.biz Developer API. https://localise.biz/api#!/
This PR contains 
* Boilerplate to define resources (tag, asset, locale etc)
* Implementation for updating tags, import/export a translation file.

**Comment by giridhar** [comment]
overall the code looks good and does what it is supposed to do, can you please add tests.

File: spec/fixtures/locales/export.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
+{
+  "addFile": "Додати файл",
```
Comment (mahesh-karipe) [inline spec/fixtures/locales/export.json:2]: can we add some fake translations instead of using application translations?

**Pull request merged**